demos/python_demos/speech_recognition_demo/README.md

```diff
@@ -16,20 +18,20 @@ After computing audio features, running a neural network to get per-frame charac
 
 The demo depends on the `ctcdecode_numpy` Python extension module,
 which implements CTC decoding in C++ for faster decoding.
-Please refer to [Using Open Model Zoo demos](../../README.md) for instructions
+Please refer to [Open Model Zoo demos](../../README.md#build-the-demo-applications) for instructions
```

Comment:
The section you linked only describes how to build the module. It doesn't explain how to prepare the environment for running the demo.

Reply:
Done. Let the user read the whole doc.